src/routers/handlers/choose_your_package_accounts/package.type.options.ts

```diff
@@ -71,12 +74,12 @@ function getHint(fee: string): string  {
 
 export function getPackageTypeOptionsRadioButtonData() {
     return packageTypeOptions.map(option => {
-        return {
+        return option.disabled === undefined || option.disabled === false ? {
```

Comment:
Instead of this could we use a filter here, so like: 
```
    return packageTypeOptions.filter(option => option.disabled !== false).map(option => {
        ...
```
That way you don't need to do the filter in choose your package accounts.